### DISCUSSION TITLE
Write files before running code

### DISCUSSION THREAD

**Discussion by anhng195** [CRAG666/code_runner.nvim#75] [Ideas]
Adding a feature that writes files before running the code would be great. This would save us time since we wouldn’t have to write the files before running the code.

**Comment by azinsharaf** [comment]
You can use `auto-save.nvim` .

```
return {
  "Pocco81/auto-save.nvim",
  enabled = true, 
  event = { "User AstroFile", "InsertEnter" },
  opts = {},
}
```

**Comment by CRAG666** [comment]
You don't need another plugin coderunner already does that with a function before
